src/TLuaInterpreterMapper.cpp

```diff
@@ -2489,6 +2497,7 @@ int TLuaInterpreter::lockRoom(lua_State* L)
     if (pR) {
         pR->isLocked = b;
         host.mpMap->setUnsaved(__func__);
+        host.mpMap->update();
```

Comment:
Locking a room does not produce any visible changes - so this is not needed.

Reply:
See https://github.com/Mudlet/Mudlet/pull/7187#discussion_r1544529665